app/models/seed_trade.rb

```diff
@@ -0,0 +1,46 @@
+class SeedTrade < ActiveRecord::Base
+  belongs_to :seed
+  belongs_to :requester, class_name: 'Member'
+
+  default_scope { order(created_at: :desc) }
+
+  # List seed trades both when member is requester and seed owner
+  scope :by_member,
+                 ->(member) { includes(:seed).where(
+                 'seeds.owner_id = ? OR requester_id = ?',
+                 member.id, member.id).references(:seeds) }
```

Comment:
We should think about performance when this table gets big. We'll probably want to make the main view default to not_replied by_member and need to decide which order to do those in and which indexes to have on the database.